index.html

```diff
@@ -151,10 +151,36 @@ <h3>Skills</h3>
                 <li>Codekit</li>
                 <li>GitHub</li>
                 <li>Codepen</li>
-                <li>Gitla</li>
+                <li>Gitlab</li>
                 <li>Terminal</li>
             </ul>
         </div>
     </section>
+    <footer>
```

Comment:
- [ ] Please, don't add the contact form within the footer. Instead, you should create a new section, and let the footer to be the end of the page with some information but not contact forms